clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesDescriptionListGroup.tsx

```diff
@@ -19,120 +13,19 @@ const ModelPropertiesDescriptionListGroup: React.FC<ModelPropertiesDescriptionLi
-      action={
-        !isArchive && (
-          <Button
-            variant="link"
-            data-testid="add-property-button"
-            icon={<PlusCircleIcon />}
-            iconPosition="start"
-            isDisabled={isAdding || isSavingEdits}
-            onClick={() => {
-              setIsShowingMoreProperties(true);
-              setIsAdding(true);
-            }}
-          >
-            Add property
-          </Button>
-        )
-      }
-      isEmpty={!isAdding && keys.length === 0}
-      contentWhenEmpty="No properties"
-    >
-      <Table aria-label="Properties table" data-testid="properties-table" variant="compact">
-        <Thead>
-          <Tr>
-            <Th>Key {isEditingSomeRow && requiredAsterisk}</Th>
-            <Th>Value {isEditingSomeRow && requiredAsterisk}</Th>
-            <Th screenReaderText="Actions" />
-          </Tr>
-        </Thead>
-        <Tbody>
-          {shownKeys.map((key) => (
-            <ModelPropertiesTableRow
-              key={key}
-              isArchive={isArchive}
-              keyValuePair={{ key, value: filteredProperties[key].string_value }}
-              allExistingKeys={allExistingKeys}
-              isEditing={editingPropertyKeys.includes(key)}
-              setIsEditing={(isEditing) => setIsEditingKey(key, isEditing)}
-              isSavingEdits={isSavingEdits}
-              setIsSavingEdits={setIsSavingEdits}
-              saveEditedProperty={(oldKey, newPair) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'update', oldKey, newPair }),
-                )
-              }
-              deleteProperty={(oldKey) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'delete', oldKey }),
-                )
-              }
-            />
-          ))}
-          {isAdding && (
-            <ModelPropertiesTableRow
-              isAddRow
-              allExistingKeys={allExistingKeys}
-              setIsEditing={setIsAdding}
-              isSavingEdits={isSavingEdits}
-              setIsSavingEdits={setIsSavingEdits}
-              saveEditedProperty={(_oldKey, newPair) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'create', newPair }),
-                )
-              }
-            />
-          )}
-        </Tbody>
-      </Table>
-      {needExpandControl && (
-        <Button
-          variant="link"
-          className={spacing.mtSm}
-          data-testid="expand-control-button"
-          onClick={() => setIsShowingMoreProperties(!isShowingMoreProperties)}
-        >
-          {isShowingMoreProperties
-            ? 'Show fewer properties'
-            : `Show ${numHiddenKeys} more ${numHiddenKeys === 1 ? 'property' : 'properties'}`}
-        </Button>
-      )}
-    </DashboardDescriptionListGroup>
-  );
-};
+}) => (
+  <DashboardDescriptionListGroup
```

Comment:
Does this need to be in a DescriptionListGroup? I think it's a separate section since you don't even need the title of it, I think we can fully use it without this component.